### DISCUSSION TITLE
How to find all QSO from defined My Gridsquare for export?

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#424] [General]
I have multiple station locations with different grid squares. Manual export to ADIF from each location is boring.
Advanced Log View does not support filter by My Gridsquare (My Refs), only filter by DX Gridsquare is supported.
Advanced Search has no corresponding field in the list to create a search request.
So there is a question: how to find all QSO from defined My Gridsquare for export?

**Comment by HB9HIL** [comment]
Hello Michael 

You can use Advanced Search for it. Just use `station_id` because the MY_GRID corresponds to the Station ID. 

To search for more then one Station ID choose `OR` instead `AND` and just add another rule with the second `station_id`

**Reply by R1BLH** [reply]
OMG. This is like the same process as exporting manually one by one from all station locations :(
Advanced log view shows My Refs column. Search result from all station locations could be sorted by this column. 
It would be good to add filter for this column.

**Reply by HB9HIL** [reply]
If you want to export one special my_grid from station locations, use the normal ADIF export (even if it's "boring"). 

If you want to export multiple my_grid from station locations, you can use Advanced Search. 

A filter for that in LBA means work for a problem, where already a solutions exists. 

Or did I get something wrong? What exactly do you try to do?

**Reply by HB9HIL** [reply]
> OMG. This is like the same process as exporting manually one by one from all station locations :( Advanced log view shows My Refs column. Search result from all station locations could be sorted by this column. It would be good to add filter for this column.

It's not the same process. You can create 1 adif file from two station locations with advanced search.

**Reply by R1BLH** [reply]
Okay, let me try to explain.
I have 78 station locations right now (thanks to eQSL). Most of them are from two grids, but there are several other grids as well.
The locations for one grid could be exported as usual.
But manually adding about 40 locations to the search query... It is not the most effective way, I think.

**Reply by HB9HIL** [reply]
Yeah I know your setup. We talked several times about that. I'm afraid but there won't be a good solution except the solutions I already told you. 

By the way you can also query all station locations which are NOT a specific one. (And you can save that query if you use it regulary.)

**Reply by R1BLH** [reply]
Oops. Understandable :)

It's all about uploading previous logs to the RDA award as an activator. I'll do it manually.
